firecloud/fiss.py

```diff
@@ -1226,12 +1228,29 @@ def mop(args):
         # Check output produces a string in Py2, Bytes in Py3, so decode if necessary
         if type(bucket_files) == bytes:
             bucket_files = bucket_files.decode()
+        
+        #List of user submission within this workspace. To ensure deletion of files in the submission directories only
+        user_submission = fapi.list_submissions(args.project, args.workspace)
```

Comment:
Please add a check here to ensure the api call succeeded (see other functions for examples of `fapi._check_response_code()`)